### DISCUSSION TITLE
Paid 5 dollar for the installer but nothing was given. Scam?

### DISCUSSION THREAD

**Discussion by Sowlinyin** [Sanster/IOPaint#326] [General]
I paid 5 dollar for the one click installer and nothing was given to me afterward. There are no emails or redirection to any pages. Is the payment an scam?

**Comment by Sanster** [comment]
Thank you for your support 🙏. The email you filled in will receive a message (if it's not in the inbox, please check the spam folder...), which contains a link to the download page.
